### PR TITLE
Add coin icons

### PR DESCRIPTION
**Pull request by andygoossens**
This pull requests add coin icons to the web UI. The most popular coins have an icon.

If you dislike all these different colors and much prefer coin icons in black/white, then you can remove the inclusion of "cryptocoins-colors.css".

Places where I did not add icons:
* Logfile (I don't want to parse that)
* P&L's "Last 24 Hours" and "All" sections (as they didn't even have an exchange icon). But note that I have added an icon on the X axis of the "Per-Coin Graph" section.
* P&L's "Daily Graph" dropdown box (as HTML doesn't allow styling option tags; and I wanted to avoid a complex workaround for now)
* Alerts (I don't want to parse that)

### CODE REVIEW

File: web/ui.js [hunk 1]
```diff
@@ -162,15 +162,25 @@ $(function() {
         return "<span class=\"" + (x < 0 ? "neg" : "pos") + "\">" + rnd8(Math.abs(x)) + "</span>";
     }
 
-    function updateGraph() {
+  function getIcon(symbol) {
```
Comment (cryptoeax) [inline web/ui.js:165]: There is some indentation issues here but I will be happy to fix these myself post merging!

**Comment by cryptoeax** [comment]
Very nice, thank you very much!

File: web/ui.js (hunk 1 above)
Comment (andygoossens) [reply web/ui.js:165]: Whoops, I did not notice that. Thanks for fixing the indentation.

Comment (cryptoeax) [reply web/ui.js:165]: NP.  I also landed a few spot fixes after the fact to fix spacing issues in the UI for missing icons, and also display icons for `SC` and `BLK`...  I suspect we'd need to canonicalize more coin names in the future.  If you had some time to look into what other coins we're consistent in, I'd appreciate that!

**Pull request merged**